usaspending_api/idvs/v2/views/amounts.py

```diff
@@ -59,11 +59,11 @@ def fetch_account_details_idv(award_id, award_id_column) -> dict:
     grandchild_award_ids = []
     child_award_ids.extend([x["award_id"] for x in children])
     grandchild_award_ids.extend([x["award_id"] for x in grandchildren])
-    award_id_sql = "faba.award_id in {award_id}".format(award_id=child_award_ids)
+    award_id_sql = "faba.award_id in {award_id}".format(award_id="(" + str(child_award_ids).strip("[]") + ")")
```

Comment:
Isn't str(<list>) = "[<list items>]" ?
is the SQL becoming `"faba.award_id in ([child_award_ids])"`

Reply:
no? I don't understand what you mean by `str()="[]"`, we turn the array into a string, strip out the `[` and `]` characters, then add parenthesis.

Reply:
updated comment to show the list

Reply:
Tested locally, it looks OK